src/main/java/com/revature/security/core/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.revature.security.core;
+
+import com.revature.security.extra.JwtTokenUtil;
+import java.io.IOException;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+/**
+ *
+ * @author FayeRedd
+ */
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+    
+    @Autowired
+    private UserDetailsService userDetailsService;
+    
+    //This is the token generator/validator. Can be replaced with future microservice providing/varifying tokens
+    @Autowired
+    private JwtTokenUtil jwtTokenUtil;
+    
+    @Value("${jwt.header}")
+    private String tokenHeader;
+    
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws ServletException, IOException {
+        String authToken = request.getHeader(this.tokenHeader);
+        
+        String username = jwtTokenUtil.getUsernameFromToken(authToken);
+        
+        if(username != null && SecurityContextHolder.getContext().getAuthentication() == null){
+            UserDetails userDetails = this.userDetailsService.loadUserByUsername(username);
+            
+            if(jwtTokenUtil.validateToken(authToken, userDetails)){
+                UsernamePasswordAuthenticationToken authentication = new UsernamePasswordAuthenticationToken(userDetails, null, userDetails.getAuthorities());
+                
+                authentication.setDetails(new WebAuthenticationDetailsSource().buildDetails(request));
+                
+ //               myLogger.info("authenticated user " + username + ", setting security context");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)